presto-hudi/src/main/java/com/facebook/presto/hudi/HudiSessionProperties.java

```diff
@@ -47,6 +48,7 @@
     private static final String STANDARD_SPLIT_WEIGHT_SIZE = "standard_split_weight_size";
     private static final String MINIMUM_ASSIGNED_SPLIT_WEIGHT = "minimum_assigned_split_weight";
     public static final String READ_MASKED_VALUE_ENABLED = "read_null_masked_parquet_encrypted_value_enabled";
+    private static final String MAX_PARTITIONS_PER_SCAN = "max-partitions-per-scan";
```

Comment:
nit: I guess better to follow the same convention of defining session properties using `_`. Let us change it to `max_partitions_per_scan`

Reply:
OK, I have changed it.